src/pages/homePage/HomePage.tsx

```diff
@@ -0,0 +1,18 @@
+import './HomePage.css';
+
+
+const HomePage: React.FC = () => {
+  return (
+    <>
+      <div className="homepage" style={{ overflow: 'hidden' }} />
+
+      <div className="home_description">
+          Velkommen til<br />
+          <img src="assets/logo.png" alt="logo"/><br />
+          Logg inn for å melde deg på aktiviteter!
```

Comment:
Would suggest to not use <br> elements to break lines.
(not so good semantic meaning for those, and harder to style. Also it's better to group your things in logical blocks)

I would suggest to use <p> instead.

Reply:
Changed to <p> instead.